projects/packages/my-jetpack/_inc/data/evaluation-recommendations/use-evaluation-recommendations.ts

```diff
@@ -14,6 +14,8 @@ import { getMyJetpackWindowInitialState } from '../utils/get-my-jetpack-window-s
 import isJetpackUserNew from '../utils/is-jetpack-user-new';
 import useWelcomeBanner from '../welcome-banner/use-welcome-banner';
 
+const NUMBER_OF_RECOMMENDATIONS_TO_SHOW = 5;
```

Comment:
Oh, thanks for the reminder. I added it as a const so we could update it in one place instead of multiple places at the same time, but `5` is an arbitrary value that we still needed to discuss about.